tests/integration/hover-completion-forward-precedence.test.ts

```
@@ -0,0 +1,335 @@
+    let test_temp_dir: string;
+    let pipeline: ReturnType<typeof build_pipeline>;
+
+    beforeEach(() => {
+        test_temp_dir = mkdtempSync(
+            join(tmpdir(), 'hover-completion-forward-precedence-'),
+        );
+        pipeline = build_pipeline();
+    });
+
+    afterEach(() => {
+        try { pipeline?.scope_resolver?.dispose(); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)